test/language/expressions/arrow-function/arrow/apply_1.js

```diff
@@ -0,0 +1,17 @@
+// Copyright 2015 Microsoft Corporation. All rights reserved.
+// This code is governed by the license found in the LICENSE file.
+
+/*---
+description: invoke arrow function using apply
+es6id: 14.2
+---*/
+
+var a = (x) => x;
+
+assert.sameValue(a.apply(null, [1]), 1, "Invocation of arrow function failed using fn.apply(null, [1]).");
+
+assert.sameValue(a.apply(undefined, [1]), 1, "Invocation of arrow function failed using fn.apply(undefined, [1]).");
+
+assert.sameValue(a.apply(this, [1]), 1, "Invocation of arrow function failed using fn.apply(this, [1]).");
+
+assert.sameValue(a.apply({}, [1]), 1, "Invocation of arrow function failed using fn.apply({}, [1]).");

```

Comment:
Duplicates

- https://github.com/tc39/test262/blob/master/test/language/expressions/arrow-function/lexical-this.js